apps/src/templates/projects/projectTypeMap.js

```diff
@@ -45,6 +45,7 @@ export const PROJECT_TYPE_MAP = {
   story: i18n.projectTypeStory(),
   time_capsule: i18n.projectTypeTimeCapsule(),
   transformers: i18n.projectTypeTransformers(),
+  music_dance_ai: 'Music Dance AI Mashup',
```

Comment:
@dju90 similarly here, we should discuss what a good user-facing translated project type should be (this is just placeholder and can be more easily updated later on by updating the translation).